clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -134,10 +134,6 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
     });
   }, [cookie, options.debug]);
 
-  const handleDismiss = useCallback(() => {
-    handleUpdatePreferences(ConsentMethod.dismiss, initialEnabledNoticeKeys);
```

Comment:
I believe we still want to keep dismissals automatically saving preferences (for now).

Reply:
putting this back